modules/channel-web/src/views/lite/main.tsx

```diff
@@ -200,6 +201,15 @@ class Web extends React.Component<MainProps> {
     }
   }
 
+  handleClearConversation = async (event: Message) => {
+    if (this.props.config.conversationId && Number(this.props.config.conversationId) !== Number(event.conversationId)) {
+      // don't do anything, it's a message from another conversation
+      return
+    }
+
+    await this.props.clearConversation()
+  }
```

Comment:
Actually, this could be in the if condition instead. Unless there's a lot of logic, it would be simpler